app/views/users/index.html.erb

```diff
@@ -1,2 +1,17 @@
-<h1>Here is a list of users</h1>
-<p>Find me in app/views/users/index.html.erb</p>
+<% @users.each do |user| %>
```

Comment:
- [ ] [OPTIONAL] instead of using `@users.each` a better practice could be using a partial to list all the users like so:
```ruby
  # app/views/users/_user.html.erb

     #   HTML Code Here displaying
     #   user.photo, user.name and user.posts_counter
```
```ruby
    # app/views/users/index.html.erb

    <%= render @users %>

```

Reply:
Thank you for the tip. I was reading the Rails' best practices and thinking about doing that.